src/main/java/frc/robot/commands/CombinedIntakeCDSForwardCommand.java

```diff
@@ -37,7 +37,7 @@ public void initialize() {
     CDSSubsystem.CDSBeltToggle(false);
     CDSSubsystem.CDSWheelToggle(false);
     intakeSubsystem.toggleIntake(false);
-    shooterSubsystem.runCargo(Constants.Shooter.cargoReverse);
+    shooterSubsystem.runCargo(Constants.reverseStopperWheelSpeed);
   }
 
   // Called every time the scheduler runs while the command is scheduled.
```

Comment:
I think you want to move the initialize stuff to execute, and then call the ball management's execute from there

something like this pseudo code
```
//Create new CDSBallManagement command object called ballManagementCmd

public void execute() {
  if (CDSSubsystem.getState() == IDLE) {
    // run CDS/intake/stopper wheel
  } else {
    ballManagementCmd.execute();
  }
``` 
I think this will work without having subsystem conflicts since we aren't using the scheduler to run it.

We might want to do something for the Outtake, but maybe not